test/views.js

```diff
@@ -16,7 +16,28 @@ const views = (function views() {
       Ask: { ask: 'What time is it?' },
     },
     ExitIntent: {
-      Farewell: { tell: 'Ok. For more info visit {site} site.' },
+      Farewell: {
+        tell: 'Ok. For more info visit {site} site.',
+        directives: {
+          type: 'Display.RenderTemplate',
+          template: {
+            type: 'BodyTemplate1',
+            backButton: 'HIDDEN',
+            backgroundImage: {
+              sources: [
```

Comment:
will this support creating the array from a variable? if so, we need to test it

`sources: '{variable}'`